lib/Translator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+// Copyright 2022. Plesk International GmbH. All rights reserved.
+
+namespace WHMCS\Module\Server\Plesk360Monitoring;
+
+class Translator
+{
+    private const DEFAULT_LANGUAGE = 'english';
```

Comment:
Does WHMCS works with `english`, `german`, ... or can we change it to ALPHA-2 like `en`, `de`? Because I think it's safer and prevents errors if someone change `german` to `deutsch`

Reply:
This is the WHMCS locale naming convention we must follow